docs/DeveloperGuide.rst

```diff
@@ -0,0 +1,182 @@
+    $ source ~/PyVENV/bin/activate
+    (PyVENV) $ git clone git://github.com/starteam/starcellbio_html
+
+
+Setup Development Environment
+-----------------------------
+
+Using PyCharm
+    View->Tool Windows->Database + Data Source -> MySQL
+    Install the database drivers if prompted
+
+
+mysqld command starts database
+
+Reference: `Installing MySQL on Mac OS X <https://rtcamp.com/tutorials/mac/osx-brew-php-mysql-nginx/>`_
+
+If you need to install MySQL on Mac OS X, execute the following command::
+
+    $ brew install mysql --enable-debugging
+
+In a terminal window execute the following commands::
+
+    $ source ~/PyVENV/bin/activate
+    (PyVENV) $ ls /usr/local/Cellar/mysql # shows foldername (a number) to use in the next command
+    (PyVENV) $ sudo cp /usr/local/Cellar/mysql/5.6.22/homebrew.mxcl.mysql.plist ~/Library/LaunchAgents/
+
+To Start mysql::
+
+    (PyVENV) $ sudo launchctl unload -w ~/Library/LaunchAgents/homebrew.mxcl.mysql.plist
+    (PyVENV) $ sudo launchctl load -w ~/Library/LaunchAgents/homebrew.mxcl.mysql.plist
+
+To Stop mysql::
+
+    (PyVENV) $ sudo launchctl unload -w ~/Library/LaunchAgents/homebrew.mxcl.mysql.plist
+
+To pip install
+
+    (PyVENV) $ cd starcellbio_html
+    (PyVENV) $ sudo pip install -r requirements.txt
+
+
+Using watch.py to compile .soy and .gss files into .js and .css files
+---------------------------------------------------------------------
+After any changes to .soy or .gss files you need to compile them into .js and .css files for your changes to work.
+
+.. note:: This repo was built before PyCharm was the default project IDE. It is now possible to create a File Watcher
+in PyCharm that will automatically run the transpilers as .soy and .gss files are changed.  This will require minor
+modifications to watch.py.
+
+In new terminal window execute the following commands::
+
+    $ source ~/PyVENV/bin/activate
+    (PyVENV) $ cd starcellbio_html
+    (PyVENV) $ sudo pip install -r requirements.txt
+
+    (PyVENV) $ export PROJECT_HOME="/Users/starcellbio/starcellbio_html" # watch.py needs repo at PROJECT_HOME
+    (PyVENV) $ cd $PROJECT_HOME/html_app # watch.py needs repo in this location
+    (PyVENV) $ python watch.py
+
+Start the StarCellBio development server on localhost
+-----------------------------------------------------
+
+To ``START`` the starcellbio_html development server in a new Terminal Window::
+
+    $ source ~/PyVENV/bin/activate
+
+.. note:: Django may not work due to an error starting mysql. The first two lines that follow get mysql up
+and running. The second two lines start the StarCellBio Django server. If you don't have mysql installed,
+you'll need to do that.
+
+
+    (PyVENV) $ cd $PROJECT_HOME
+
+    (PyVENV) $ sudo ./manage.py collectstatic
+    (PyVENV) $ sudo ./manage.py loaddata backend statuses courses assignments studentassignments
+    (PyVENV) $ sudo ./manage.py runserver
+
+./manage.py loaddata backend statuses courses assignments studentassignments
```

Comment:
After all that, I have it up and running, and it looks like:
![image](https://cloud.githubusercontent.com/assets/596029/5828515/3b14cf86-a0d3-11e4-94bd-c4d70ff3f454.png)

which seems right, but I really have no idea.